ledger/store/trackerdb/version.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2019-2023 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package trackerdb
+
+// AccountDBVersion is the database version that this binary would know how to support and how to upgrade to.
+// details about the content of each of the versions can be found in the upgrade functions upgradeDatabaseSchemaXXXX
+// and their descriptions.
+var AccountDBVersion = int32(9)
```

Comment:
not sure if it makes sense to have a single file for AccountDBVersion that is used only in schema/migrations.

Reply:
it doesn't really belong on any of the other files as it is right now. 

I don't particularly like it being exposed on the public interface.
It carries internal meaning of the database schema. 
We should likely have hidden versions inside each of the impl folders that are specific to the db in question. 
It is currently used in tests outside the store package. 

We don't have interface statics or constants either for it to be part of the `TrackerDb` interface which is where it should be.